pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -298,9 +305,51 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 		return err
 	}
 
+	if err := o.getAPIBindings(ctx, newServerHost); err != nil {
+		// display the error, but don't stop the current workspace from being reported.
+		_, _ = fmt.Fprintf(o.Out, "error checking APIBindings: %v", err)
+	}
+
 	return currentWorkspace(o.Out, newServerHost, shortWorkspaceOutput(o.ShortWorkspaceOutput), workspaceType)
 }
 
+// getAPIBindings retrieves APIBindings within the workspace and look for ones without exported PermissionClaims.
+func (o UseWorkspaceOptions) getAPIBindings(ctx context.Context, host string) error {
```

Comment:
```suggestion
func (o *UseWorkspaceOptions) getAPIBindings(ctx context.Context, host string) error {
```

Reply:
Also, I'd either rename this to something like `checkForAPIBindingsWithUnresolvedClaims`, or split it up so this actually gets bindings, and another function does the checking.